Tests/Fluent.Tests/WebApp/WebAppsMsiTests.cs

```diff
@@ -93,7 +93,7 @@ public void CanCRUDWebAppWithMsi()
                     {
                         UploadFileToWebApp(webApp.GetPublishingProfile(), Path.Combine(".", "Assets", "appservicemsi.war"));
 
-                        SdkContext.DelayProvider.Delay(10000);
+                        SdkContext.DelayProvider.Delay(30000);
```

Comment:
Why change this?

Reply:
10 sec was too short for the upload operation. Make it delay longer should make sense.